spectests/structs_encoding.go

```diff
@@ -5283,7 +5283,7 @@ func (b *BeaconStateAltair) HashTreeRootWith(hh ssz.HashWalker) (err error) {
 			err = ssz.ErrIncorrectListSize
 			return
 		}
-		hh.PutBytes(b.PreviousEpochParticipation)
+		hh.Append(b.PreviousEpochParticipation)
 		hh.MerkleizeWithMixin(elemIndx, byteLen, (1099511627776+31)/32)
```

Comment:
Prevents double merkleization of `PutBytes` and `MerkleizeWithMixin` used together.